sxt/multiexp/bucket_method/BUILD

```diff
@@ -18,6 +18,38 @@ sxt_cc_component(
     ],
 )
 
+sxt_cc_component(
+    name = "accumulation",
+    test_deps = [
+        "//sxt/memory/resource:managed_device_resource",
+        "//sxt/base/device:synchronization",
+        "//sxt/base/curve:example_element",
+        "//sxt/execution/schedule:scheduler",
+        "//sxt/base/test:unit_test",
```

Comment:
Out of order includes.

Reply:
Fixed.

We should probably add that to the format script.

Reply:
Agreed. Maybe we can use [Buildifier](https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md)?